### PR TITLE
Data viewers

### PR DESCRIPTION
**Pull request by JSiuda1**

### CODE REVIEW

File: src/data_displays/displays/data_display_plot.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
+import json
+from typing import Any, Self
+from PySide6.QtWidgets import QGridLayout
+
+from src.data_displays import DataDisplayBasic
+from src.data_displays.plot.data_plot import DataPlot, logger
+
+
+class DataDisplayPlot(DataDisplayBasic):
```
Comment (kgliwinski) [inline src/data_displays/displays/data_display_plot.py:9]: are these plots auto range?

Comment (kgliwinski) [reply src/data_displays/displays/data_display_plot.py:9]: aah ok the ranges are defined in data_test_*

**Review by kgliwinski — APPROVED** [review]
lgtm 😮‍💨

File: config/data_text.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    {
+      "name": "title"
+    },
+    {
+      "name": "pressure",
+      "lower_bound": 0,
+      "upper_bound": 10
+    },
+    {
+      "name": "valve_state",
```
[the remark this answers is not in the document]
Comment (JSiuda1) [reply config/data_text.json:14]: The json configs will be udpated during the developement, now some dummy values was generated by copilot 👮‍♂️

**Pull request merged**